src/ImageGallery.jsx

```diff
@@ -1271,9 +1271,9 @@ export default class ImageGallery extends React.Component {
               onError={handleImageError}
             >
               {
-                item.imageSet.map(source => (
+                item.imageSet.map((source, index) => (
```

Comment:
IMHO, you should use something else then `index`.

> The key is used by React to identify which items have changed, are added, or are removed and should be stable.
see https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

> (...) We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. 
see https://reactjs.org/docs/lists-and-keys.html#keys.

Check out how @xiaolin resolve this previously: https://github.com/ahakem/react-image-gallery/blob/master/src/ImageGallery.jsx#L625

Reply:
you can also fix this by using a different `source.media` for example if you need the same `source.media` add a dummy query string e.g. `source.com/media` and `source.com/media?dummy=1`

Reply:
I've updated the key so now we have the source media beside the index, so it's not the index and at the same time the key will never be duplicated.

Reply:
Can you update the rest of the keys to do the same? This way it is consistent with the other keys.

Reply:
@xiaolin could you explain more, did not got what you mean.

Reply:
https://github.com/xiaolin/react-image-gallery/blob/master/src/ImageGallery.jsx#L625
https://github.com/xiaolin/react-image-gallery/blob/master/src/ImageGallery.jsx#L660
https://github.com/xiaolin/react-image-gallery/blob/master/src/ImageGallery.jsx#L693

If you can update these to also have an index at the end it would make the keys more consistent.

Reply:
And also change `key={`${source.media}-${index}`}` => `key={`media-${source.media}-${index}`}`

Reply:
updated.